src/Android/Xamarin.Android/Samples/Search/FindPlace/FindPlace.cs

```diff
@@ -421,19 +421,19 @@ private void ShowStatusMessage(string message)
             string searchText = _myLocationBox.Text;
 
             // Get the results
-            IEnumerable<String> results = await GetSuggestResults(searchText);
+            List<string> results = (await GetSuggestResults(searchText)).ToList();
```

Comment:
Why not just return a List from GetSuggestResults, like we did in UWP?

Reply:
Good catch, I meant to update all the platforms to be consistent but missed some.